_posts/2025-09-05-musical-physics-poster.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Musical Physics - Lessons from building music apps in Unity
+author: Pete Nancollis, Ant Nasce, Tom Cartwright
+---
+
+<img src="/assets/ADC_banner.png" style="border-radius: 15px" />
+
+We used the Unity game engine to create Playtonik - an experimental instrument, midi device and fidget toy for iOS.
+
+We chose to use a game engine because it was the fastest place to test to the novel feature of our app: the interaction of physics objects to make sounds.
+
+## Why use a Game Engine?
+
+### Physics as a playground of sound
+
+Simulations of interacting objects are a fantastic source of sound triggers, allowing for everything from a natural, real feel to something more abstract and chaotic.
```

Comment:
Simulations of interacting objects are a fantastic source of sound triggers, allowing for everything from a natural "real feel", to something more abstract and chaotic.